### PR TITLE
Clean up - Updated spec according build service

### PR DESCRIPTION
**Pull request by mchf**

### CODE REVIEW

File: package/yast2-network.spec [hunk 1]
```diff
@@ -23,13 +23,13 @@ Release:        0
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 Source0:        %{name}-%{version}.tar.bz2
 
-Group:          System/YaST
-License:        GPL-2.0
 # should be required by devtools
-BuildRequires:  perl-XML-Writer pkgconfig rpm
+BuildRequires:  perl-XML-Writer
+BuildRequires:  pkgconfig
```
Comment (jreidinger) [inline package/yast2-network.spec]: why it is needed? I do not see any pkgconfig usage

File: package/yast2-network.spec [hunk 2]
```diff
@@ -23,13 +23,13 @@ Release:        0
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 Source0:        %{name}-%{version}.tar.bz2
 
-Group:          System/YaST
-License:        GPL-2.0
 # should be required by devtools
-BuildRequires:  perl-XML-Writer pkgconfig rpm
+BuildRequires:  perl-XML-Writer
```
Comment (jreidinger) [inline package/yast2-network.spec]: I think it can be dropped

File: package/yast2-network.spec [hunk 3]
```diff
@@ -23,13 +23,13 @@ Release:        0
...
+BuildRequires:  pkgconfig
+BuildRequires:  rpm
```
Comment (jreidinger) [inline package/yast2-network.spec]: useless, can be removed

**Comment by mchf** [comment]
In general I have no idea about those requires you highlighted. It is just what I got back from BS ;-) I'll update it.

**Comment by jreidinger** [comment]
@mchf LGTM, in general if you remove BuildRequire and still can build package, then it is not needed :wink:

File: package/yast2-network.spec [hunk 4]
```diff
@@ -23,13 +23,10 @@ Release:        0
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 Source0:        %{name}-%{version}.tar.bz2
 
-Group:          System/YaST
-License:        GPL-2.0
 # should be required by devtools
-BuildRequires:  perl-XML-Writer pkgconfig rpm
```
Comment (mvidner) [inline package/yast2-network.spec]: Sorry, now you're mixing up formatting changes done by `/usr/lib/obs/service/format_spec_file.files/prepare_spec` with semantic changes to the dependencies. I do not care if we do either of them, but I do care to not mix them in a single commit.

File: package/yast2-network.spec [hunk 5]
```diff
@@ -24,9 +24,6 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 Source0:        %{name}-%{version}.tar.bz2
 
 # should be required by devtools
```
Comment (mvidner) [inline package/yast2-network.spec]: The comment belongs to the deleted lines, delete it too.

**Comment by mchf** [comment]
thanks for reviews

**Pull request merged**